src/loom/alg.clj

```diff
@@ -432,9 +432,9 @@ can use these functions."
                       color (- 1 (coloring v))
                       nbrs (graph/successors g v)]
                   ;; TODO: could be better
-                  (if (some #(and (coloring %) (= (coloring v) (coloring %)))
+                  (when-not (some #(and (coloring %) (= (coloring v) (coloring %)))
                             nbrs)
-                    nil ;not bipartite
+                    ;not bipartite
```

Comment:
In this case, we should leave the if-statement to make it clear that the true branch of the if-statement, which returns nil, indicates that the graph is not bipartite.